Week-1/IshaanG/trackprice.py

```diff
@@ -0,0 +1,42 @@
+import spec
+import time
+import smtplib
+import ssl
+
+
+class TrackPrice:
+    def __init__(self):
+        self.config = spec.Spec().read_config()
+        self.current_price = self.getCurrentPrice()
+
+    def getCurrentPrice(self):
+        specs = spec.Spec()
```

Comment:
How about sticking to just one: snake_case or camelCase?

I'd prefer snake_case: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names